### PR TITLE
add new paragraph in about page

### PR DESCRIPTION
**Pull request by Benouz6**
Here is the new paragraph

### CODE REVIEW

**Review by stephlaf — CHANGES_REQUESTED** [review]
Good job waiting for that tiny change before merging

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 <h1>This is about the team</h1>
 <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Beatae quis aut deserunt ex ab assumenda corporis officiis cumque exercitationem velit. Maiores earum itaque minus molestias quo, eveniet adipisci ipsa ad!</p>
+<p>add a new paragraph</p>
```
[the remark this answers is not in the document]
Comment (Benouz6) [reply app/views/pages/about.html.erb]: @stephlaf thanks

[the remark this answers is not in the document]
Comment (Benouz6) [reply app/views/pages/about.html.erb]: @stephlaf  change is done

**Review by stephlaf — APPROVED** [review]
Good job 🎉

**Pull request merged**